code-challenges.js

```diff
@@ -13,52 +13,92 @@ var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
 
+const divisible = (num) => {
+    if(num % 3 === 0){
+    return num + " is divisible by three"
+} else if (num % 3 >= 1){
+    return num + " is not divisible by three"
+} else if (num % 3 >= -3){
+    return num + " is not divisible by three"
```

Comment:
this logic does work but it may be more concise to have a single `else if` that says: `(num % 3 != 0)` to cover the cases that might his these other conditions ^^